### ISSUE TITLE
Error Encountered While Creating Airflow Pool on Google Composer 2 with Airflow v2.4.3 

### ISSUE THREAD

**Issue by kadero** [terraform-google-modules/terraform-google-composer#80]
### TL;DR

I am attempting to create an Airflow pool within a Google Composer 2 environment, operating with Airflow version 2.4.3. During this process, I initiate the 'terraform apply' command, but am encountering an error.

### Expected behavior

The terraform applu works and the Airflow pool created:

It works with following command:
gcloud composer environments run --location=europe-west4 unifai-composer pools -- set "pool_training_cpu" 10 "pool training cpu" 
kubeconfig entry generated for europe-west4-unifai-compose-13964203-gke.
Executing within the following Kubernetes cluster namespace: composer-2-1-13-airflow-2-4-3-13964203
Pool pool_training_cpu created

### Observed behavior

│ Error: local-exec provisioner error
│ 
│   with module.pool_training_gpu.module.gcloud.null_resource.run_command[0],
│   on .terraform/modules/pool_training_gpu.gcloud/main.tf line 231, in resource "null_resource" "run_command":
│  231:   provisioner "local-exec" {
│ 
│ Error running command 'PATH=/google-cloud-sdk/bin:$PATH
│ gcloud composer environments run --project=unifai-ml --location=europe-west4 unifai-composer pool -- --set "pool_training_gpu" 4 "pool training gpu"
│ ': exit status 1. Output: kubeconfig entry generated for europe-west4-unifai-compose-13964203-gke.
│ ERROR: (gcloud.composer.environments.run) The subcommand "pool" is not supported for Composer environments with Airflow version 2.4.3.

### Terraform Configuration

```hcl
module "pool_training_gpu" {
  source            = "terraform-google-modules/composer/google//modules/airflow_pool"
  project_id        = data.google_client_config.config.project
  composer_env_name = module.composer.composer_env_name
  region            = data.google_client_config.config.region
  pool_name         = "pool_training_gpu"
  slot_count        = 4
  description       = "pool training gpu"
}
```

### Terraform Version

```sh
Terraform v1.1.9
on linux_amd64
+ provider registry.terraform.io/hashicorp/external v2.3.1
+ provider registry.terraform.io/hashicorp/google v4.31.0
+ provider registry.terraform.io/hashicorp/google-beta v4.27.0
+ provider registry.terraform.io/hashicorp/null v3.2.1
+ provider registry.terraform.io/hashicorp/random v3.5.1

Your version of Terraform is out of date! The latest version
is 1.4.6. You can update by downloading from https://www.terraform.io/downloads.html
```

### Additional information

Seems like the command executed by Terraform is wrong:
gcloud composer environments run --project=unifai-ml --location=europe-west4 unifai-composer pool -- --set "pool_training_gpu" 4 "pool training gpu"
instead of:
gcloud composer environments run --location=europe-west4 unifai-composer pools -- set "pool_training_cpu" 10 "pool training cpu" 

I can contribute if needed :)

Thank you :pray: !
